src/transformers/models/auto/tokenization_auto.py

```diff
@@ -213,11 +210,34 @@
     ]
 )
 
+# For tokenizers that do not have a config
+NO_CONFIG_TOKENIZER = [
+    BertJapaneseTokenizer,
+    BertweetTokenizer,
+    HerbertTokenizer,
+    HerbertTokenizerFast,
+    PhobertTokenizer,
+]
+
+
 SLOW_TOKENIZER_MAPPING = {
     k: (v[0] if v[0] is not None else v[1])
     for k, v in TOKENIZER_MAPPING.items()
     if (v[0] is not None or v[1] is not None)
 }
+SLOW_TOKENIZER_MAPPING[HerbertTokenizer] = HerbertTokenizerFast
```

Comment:
This was not in `TOKENIZER_MAPPING.items()` so not set.

Reply:
As seen offline, probably unnecessary.